### ISSUE TITLE
Problem in conformance with three missing  statements

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#454]
In the conformance the third and fourth example has problems.
In the third one it says:
<p data-localization-id="conformance-details">Detailed Conformance Information</p>
						<p>This publication reports (Publisher's in-house specification.)</p>
<p> The publication was certified
							on 2021-09-07 by Foo's Accessibility Testing with a credential of "Enterprise Accessibility
							Rating." For more information refer to the <a
								href="http://www.example.com/a11y/report/9780000000001">certifier's report.</a></p>

First, the publisher in-house statement does not have an ID. I cannot find in the techniques a way to pull out this information. The certification information I think can be extracted, but not sure how.

Next we have:
Missing a conformance statement we have:

<p data-localization-id="conformance-no" data-localization-mode="compact">The publication does not include a conformance statement</p>
						<p>Detailed Conformance Information:</p>
						<p>The conformance metadata was not found</p>

We could just repeat the previous statement, but the logic would generate the details heading.

Does this need to be resolved before our next draft?
